enforcer/src/main/java/org/wso2/choreo/connect/enforcer/util/FilterUtils.java

```diff
@@ -305,14 +309,16 @@ public static AuthenticationContext generateAuthenticationContext(String tokenId
         } else {
             authContext.setKeyType(APIConstants.API_KEY_TYPE_PRODUCTION);
         }
-
         authContext.setApiTier(apiLevelPolicy);
         if (api != null) {
             authContext.setTier(APIConstants.UNLIMITED_TIER);
             authContext.setApiName(api.getAsString(APIConstants.JwtTokenConstants.API_NAME));
             authContext.setApiPublisher(api.getAsString(APIConstants.JwtTokenConstants.API_PUBLISHER));
 
         }
+        if (!StringUtils.isEmpty(apiUUID)) {
```

Comment:
Can this UUID be empty if there is an actual matched API is found

Reply:
When the apictl is used, uuid becomes empty